Content/src/Client/style.scss

```diff
@@ -1,3 +1,24 @@
+@charset "utf-8";
+
+//#if (layout != "none")
+// FontAwesome
+$fa_font_path: "~@fortawesome/fontawesome-free/webfonts";
```

Comment:
Is it normal to load both FontAwesome and OpenSans icons at the same time?

Reply:
OpenSans is not for icons, it is for the overall fonts, at least that's what I understood.

Reply:
I believe I added open sans there as some of the fulma templates used it

Reply:
Then we could remove it as well.

Reply:
Sorry I meant we still use it when you specify `--layout fulma-hero` for example. So don't remove it for now

Reply:
But here you include it for all the layouts because it's `(layout != "none")` no?

Reply:
The condition is not perfectly valid here, currently layout can be one of: 
* none
* fulma-basic
* fulma-hero
* ... (other fulma templates)

Not sure here but probably fulma-basic doesn't need the font, so it would be better to have `layout != "none" && layout != "fulma-basic"` but I was too lazy for that

Reply:
End game is we want to eventually leave just none and fulma-basic so we could remove the font then